src/app/queryrow.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="api-query" (click)="handleQueryClick()" onclick="this.blur();" (keydown)="queryKeyDown($event)" [attr.title]="getTitle()"
-    [class.restrict]="!isAuthenticated() && query.method != 'GET'" tabindex="0">
+    [class.restrict]="!isAuthenticated() && query.method != 'GET'" tabindex="0" aria-labelledby="row-1" role="button">
```

Comment:
The aria-labelledby="row-1" makes no sense since the selection is made by element ID. Remove this, it does nothing.